<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBar" name="titleWidget" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="Login" name="loginWidget"/>
     <widget class="ResetPassword" name="resetWidget"/>
     <widget class="Registered" name="registeredWidget"/>
     <widget class="PersonalCenter" name="personalCenterWidget"/>
     <widget class="StackRoom" name="stackRoomWidget"/>
     <widget class="Manager" name="managerWidget"/>
     <widget class="Reader" name="readerWidget"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border: none;
	color: #ffffff;
	font: &quot;微软雅黑&quot;;
}

#widget
{
	border-image: url(:/Images/bar.jpg);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>0000/0/0 dddd
   hh:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StackRoom</class>
   <extends>QWidget</extends>
   <header>stackroom.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PersonalCenter</class>
   <extends>QWidget</extends>
   <header>personalcenter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Login</class>
   <extends>QWidget</extends>
   <header>login.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Registered</class>
   <extends>QWidget</extends>
   <header>registered.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Manager</class>
   <extends>QWidget</extends>
   <header>manager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Reader</class>
   <extends>QWidget</extends>
   <header>reader.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResetPassword</class>
   <extends>QWidget</extends>
   <header>resetpassword.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
